### DISCUSSION TITLE
Choices in status not start with capital letter after edit

### DISCUSSION THREAD

**Discussion by yckwan** [directus/directus#8418] [Archive]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

![image](https://user-images.githubusercontent.com/52000447/135243026-8f0faeb4-76a6-4b0d-b2cb-22113ecfab9d.png)
![image](https://user-images.githubusercontent.com/52000447/135243114-16207368-9c97-4862-8b8d-e8496a867b52.png)

### To Reproduce

1) create a collection with status.
2) modify status in choices.
3) expand status in the collection.

### What version of Directus are you using?

latest

### What version of Node.js are you using?

-

### What database are you using?

postgres latest

### What browser are you using?

edge

### What operating system are you using?

Windows WSL v2 with ubuntu

### How are you deploying Directus?

docker

**Comment by azrikahar** [comment]
Technically we will still need to type the Text and Value like this:

![chrome_3Jdtq0djBj](https://user-images.githubusercontent.com/42867097/135246195-6d12514a-e462-44bc-aafc-51ddb0c25c7c.png)

So that it will be properly capitalized:

![chrome_tS6a4PxUMt](https://user-images.githubusercontent.com/42867097/135246365-1ccd0eca-7a21-4dcb-b174-7e1a77806af3.png)

---

As for the `$t:draft` and `$t:archived`, it's a translation issue currently being addressed #8104. You don't really need to follow the `$t:` prefix to insert your own custom text/value 👍

**Reply by yckwan** [reply]
Ohhh, I see.
Thanks for information.

**Reply by yckwan** [reply]
![image](https://user-images.githubusercontent.com/52000447/135249379-2c14de94-c0c5-4ca7-aead-b05a21f613b6.png)
It works!
